### PR TITLE
Resolves #121

### LINKED ISSUE #121: Very high threshold missing
Since migration to database structure, the very high threshold is no longer being drawn on the chart.

### PR DESCRIPTION
**Pull request by xmnlab**

### CODE REVIEW

File: fludashboard/libs/views.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def data__weekly_incidence_curve(
         pass
 
     # cheating: using a new field corredor_muito_alto just for plotting
-    df['typical_very_high'] = df.very_high_threshold.max() * 1.02
```
Comment (marfcg) [inline fludashboard/libs/views.py:183]: The additional columns will guarantee that it is always above any other curve in the chart:

    df['typical_very_high'] = df[['very_high_threshold', 'ci_upper', 'value', 'typical_high']].max().max() * 1.1

**Review by marfcg — CHANGES_REQUESTED** [review]
For the typical_very_high cheat, the very_high_threshold is not sufficient to guarantee that typical_very_high will be higher than any other curve.
Example given: when the notifications crosses the very_high_threshold.

**Comment by marfcg** [comment]
It would also be good to set the yaxis upper limit to y_max = typical_very_high, with the proposed change.

File: fludashboard/libs/views.py (hunk 1 above)
Comment (xmnlab) [reply fludashboard/libs/views.py:183]: done :)

**Review by marfcg — APPROVED** [review]

**Comment by xmnlab** [comment]
> It would also be good to set the yaxis upper limit to y_max = typical_very_high, with the proposed change.

maybe there is no easy trick to do this now .. maybe you can open a new issue about this topic

**Comment by marfcg** [comment]
No problem. As long as the background is always above, we are good.
I'll open it as low priority enhancement.

**Pull request merged**